Fix Javadoc build step with Java 11

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project neo4j-jdbc: MavenReportException: Error while creating archive: 
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/7/docs/api/ are in the unnamed module.
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jPreparedStatement.java:90: warning: no @throws for java.sql.SQLException
[ERROR] 	protected void checkParamsNumber(int parameterIndex) throws SQLException {
[ERROR] 	               ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java:276: warning: no description for @param
[ERROR] 	 * @param types
[ERROR] 	   ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java:277: warning: no description for @return
[ERROR] 	 * @return
[ERROR] 	   ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java:278: warning: no description for @throws
[ERROR] 	 * @throws SQLException
[ERROR] 	   ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java:319: warning: no description for @return
[ERROR] 	 * @return
[ERROR] 	   ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java:320: warning: no description for @throws
[ERROR] 	 * @throws SQLException
[ERROR] 	   ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/utils/ObjectConverter.java:67: warning: no @param for <T>
[ERROR]     public static <T> T convert(Object from, Class<T> to) {
[ERROR]                         ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/utils/ObjectConverter.java:106: warning: no @param for value
[ERROR]     public static Map anyToMap(Object value) {
[ERROR]                       ^
[ERROR] /Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/src/main/java/org/neo4j/jdbc/utils/ObjectConverter.java:106: warning: no @return
[ERROR]     public static Map anyToMap(Object value) {
[ERROR]                       ^
[ERROR] 
[ERROR] Command line was: /Users/mh/.sdkman/candidates/java/11.0.6-zulu/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/Users/mh/d/java/neo/neo4j-jdbc/neo4j-jdbc/target/apidocs' dir.
```

--- a/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java
+++ b/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java
@@ -48,7 +48,7 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	private static final Pattern VERSION_REGEX = Pattern.compile("^(\\d+)\\.(\\d+)(\\.|-)?(.*)?$");
 
 	protected static final int PROPERTY_SAMPLE_SIZE = 1000;
-	
+
 	/**
 	 * Name of the driver.
 	 */
@@ -68,12 +68,12 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	 * Database labels.
 	 */
 	protected List<Table> databaseLabels;
-	
+
 	/**
 	 * Database keys.
 	 */
 	protected List<Column> databaseProperties;
-	
+
 	/**
 	 * The JDBC connection.
 	 */
@@ -100,7 +100,7 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 			this.driverVersion = "Unknown";
 			throw new Neo4jJdbcRuntimeException(e);
 		}
-		
+
 		this.databaseLabels = new ArrayList<>();
 		this.databaseProperties = new ArrayList<>();
 	}
@@ -273,9 +273,9 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	 * @param catalog works only with null, because #getCatalogs() return no rows
 	 * @param schemaPattern works only with null, because #getSchemas() return no rows
 	 * @param tableNamePattern works with % too
-	 * @param types
-	 * @return
-	 * @throws SQLException
+	 * @param types list of types
+	 * @return the list of related tables as ResultSet
+	 * @throws SQLException no exception is thrown by this method
 	 */
 	@Override public ResultSet getTables(String catalog, String schemaPattern, String tableNamePattern, String[] types) throws SQLException {
 		if (this.databaseLabels == null || this.databaseLabels.isEmpty()) {
@@ -316,8 +316,8 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	 * @param schemaPattern works only with null, because #getSchemas() return no rows
 	 * @param tableNamePattern works with % too
 	 * @param columnNamePattern works with % too
-	 * @return
-	 * @throws SQLException
+	 * @return the list of related columns as ResultSet
+	 * @throws SQLException no exception is thrown by this method
 	 */
 	@Override public ResultSet getColumns(String catalog, String schemaPattern, String tableNamePattern, String columnNamePattern) throws SQLException {
 		if (this.databaseProperties == null || this.databaseProperties.isEmpty()) {
@@ -404,19 +404,19 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	@Override public boolean allProceduresAreCallable() throws SQLException {
 		return true;
 	}
-	
+
 	@Override public boolean allTablesAreSelectable() throws SQLException {
 		return true;
 	}
-	
+
 	@Override public boolean nullsAreSortedHigh() throws SQLException {
 		return false;
 	}
-	
+
 	@Override public boolean nullsAreSortedLow() throws SQLException {
 		return true;
 	}
-	
+
 	@Override public boolean nullsAreSortedAtStart() throws SQLException {
 		return false;
 	}
@@ -424,7 +424,7 @@ public abstract class Neo4jDatabaseMetaData implements java.sql.DatabaseMetaData
 	@Override public boolean nullsAreSortedAtEnd() throws SQLException {
 		return true;
 	}
-	
+
 	/*---------------------------------*/
 	/*       Not implemented yet       */
 	/*---------------------------------*/

--- a/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jPreparedStatement.java
+++ b/neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jPreparedStatement.java
@@ -86,6 +86,7 @@ public abstract class Neo4jPreparedStatement extends Neo4jStatement implements P
 	 * If its is we throw an exception.
 	 *
 	 * @param parameterIndex The index parameter to check
+	 * @throws SQLException in case of the input index does not exists
 	 */
 	protected void checkParamsNumber(int parameterIndex) throws SQLException {
 		if (parameterIndex > this.parametersNumber) {
@@ -98,6 +99,7 @@ public abstract class Neo4jPreparedStatement extends Neo4jStatement implements P
 	 * If it's not we throw an exception.
 	 *
 	 * @param obj The object to check
+	 * @throws SQLException in case the object type is not supported
 	 */
 	private void checkValidObject(Object obj) throws SQLException {
 		// TODO: this may belong into org.neo4j.driver.v1.Values

--- a/neo4j-jdbc/src/main/java/org/neo4j/jdbc/utils/ObjectConverter.java
+++ b/neo4j-jdbc/src/main/java/org/neo4j/jdbc/utils/ObjectConverter.java
@@ -56,6 +56,7 @@ public class ObjectConverter {
     /**
      * Convert the given object value to the given class.
      *
+     * @param <T> the destination type of the conversion
      * @param from The object value to be converted.
      * @param to   The type class which the given object should be converted to.
      * @return The converted object value.
@@ -101,7 +102,9 @@ public class ObjectConverter {
 
     // Converters ---------------------------------------------------------------------------------
     /**
-     * Generic converter for maps.
+     * Cast an object to a map
+     * @param value the object to convert
+     * @return the converted Map
      */
     public static Map anyToMap(Object value) {
         return (Map) value;

--- a/pom.xml
+++ b/pom.xml
@@ -284,11 +284,9 @@
 						</goals>
 					</execution>
 				</executions>
-                                <configuration>
-                                        <additionalparam>-Xdoclint:none</additionalparam>
-                                        <additionalOptions>-Xdoclint:none</additionalOptions>
-                                        <additionalJOption>-Xdoclint:none</additionalJOption>
-                                </configuration>
+				<configuration>
+					<source>8</source>
+				</configuration>
 			</plugin>
 		</plugins>
 	</build>

